### DISCUSSION TITLE
Can we use react-page for on page text editing.

### DISCUSSION THREAD

**Discussion by bharat2913** [react-page/react-page#1093] [Q&A]
what I am asking is,

can we customize it to remove the side bar and we make a custom button which add react-page text(slate) plugin.

**Comment by zehawki** [comment]
If I understand you correctly - react-page is not meant for that usecase. You would be better served with a "general" RTE like Quilljs, Slate, TUI and many others which you can integrate on page, and then switch between a view and edit of a text area with a "edit" button etc.

**Reply by macrozone** [reply]
that does not answer the question.

**Reply by bharat2913** [reply]
Thanks @zehawki 
after a bit search, I've gone with [react-draft-wysiwyg](https://www.npmjs.com/package/react-draft-wysiwyg) editor.
It seems good

**Comment by macrozone** [comment]
@bharat2913 actually yes, there is an example: https://react-page.github.io/examples/customEditorChildren

source code https://github.com/react-page/react-page/blob/master/examples/pages/examples/customEditorChildren.tsx

**Reply by bharat2913** [reply]
Hey @macrozone 
Thanks for these, I'll try this, When I use this is any  project.
But for this project, after a bit search, I've gone with [react-draft-wysiwyg](https://www.npmjs.com/package/react-draft-wysiwyg) editor.
